satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -732,6 +958,19 @@ def test_excs(self, reader_configs, fh_param):
                 make_dataid(name="ir_123", calibration="unknown"),
                 {"units": "unknown"})
 
+    @pytest.mark.parametrize("channel",list_total_channel)
+    @pytest.mark.parametrize("resolution",list_resolution)
+    def test_excs_af(self,FakeFCIFileHandlerAF_fixture,reader_configs,channel):
+        """Test exceptions for AF files."""
+        fh_param = FakeFCIFileHandlerAF_fixture
+        reader = _get_reader_with_filehandlers(fh_param["filenames"], reader_configs)
+        with pytest.raises(ValueError, match="Unknown dataset key, not a channel, quality or auxiliary data: invalid"):
+            reader.file_handlers[fh_param["filetype"]][0].get_dataset(make_dataid(name="invalid"), {})
+        with pytest.raises(ValueError, match="unknown invalid value for <enum 'calibration'>"):
+            reader.file_handlers[fh_param["filetype"]][0].get_dataset(
+                make_dataid(name=f"{channel}", calibration="unknown"),
+                {"units": "unknown"})
+
```

Comment:
same as above, this is not necessary 
```suggestion
    @pytest.mark.parametrize("channel",list_total_channel)
    @pytest.mark.parametrize("resolution",list_resolution)
    def test_excs_af(self,FakeFCIFileHandlerAF_fixture,reader_configs,channel):
        """Test exceptions for AF files."""
        fh_param = FakeFCIFileHandlerAF_fixture
        reader = _get_reader_with_filehandlers(fh_param["filenames"], reader_configs)
        with pytest.raises(ValueError, match="Unknown dataset key, not a channel, quality or auxiliary data: invalid"):
            reader.file_handlers[fh_param["filetype"]][0].get_dataset(make_dataid(name="invalid"), {})
        with pytest.raises(ValueError, match="unknown invalid value for <enum 'calibration'>"):
            reader.file_handlers[fh_param["filetype"]][0].get_dataset(
                make_dataid(name=f"{channel}", calibration="unknown"),
                {"units": "unknown"})

```